test/src/test/java/org/axonframework/test/aggregate/FixtureTest_Annotated.java

```diff
@@ -37,17 +39,21 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
 
 /**
  * @author Allard Buijze
  */
 public class FixtureTest_Annotated {
 
     private FixtureConfiguration<AnnotatedAggregate> fixture;
+    private CommandTargetResolver mockCommandTargetResolver;
```

Comment:
Nit: Would you mind making this a `testAggregateIdentifier_CustomTargetResolver` test function field instead of a global field? The other test functions do not necessarily have to access this object.

Reply:
👍

Reply:
Done.